### PR TITLE
add blocks templates

### PR DESCRIPTION
**Pull request by basilkot**

### CODE REVIEW

**Comment by mvktsk** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-4517 has been created

File: config/settings_data.json [hunk 1]
```diff
@@ -1,701 +1,118 @@
...
-        "currency.code"
-      ],
-      "drafts_grid_fields": [
-        "name",
-        "itemsCount",
-        "subTotal.formattedAmount",
-        "total.formattedAmount",
-        "actions"
-      ]
+    "current": {
```
Comment (asvishnyakov) [inline config/settings_data.json:2]: Is it ok what all presets are removed?

File: assets/pages.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.list-items {
```
Comment (asvishnyakov) [inline assets/pages.css:1]: Why we need this?

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

File: config/settings_data.json (hunk 1 above)
Comment (asvishnyakov) [reply config/settings_data.json]: Fixed

File: assets/pages.css (hunk 2 above)
Comment (asvishnyakov) [reply assets/pages.css:1]: Postponed till next planning

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**